src/main/java/com/epam/ta/reportportal/database/dao/TestItemRepositoryCustom.java

```diff
@@ -208,6 +208,8 @@
 	 */
 	List<TestItem> findTestItemWithIssues(String launchId);
 
+	boolean hasChildrenWithStatuses(String itemId, Status... status);
```

Comment:
Do not forget javadoc. Whether we look for all statuses or at least one from array